pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -587,7 +623,7 @@ func (e *eksOperatorController) getRestConfig(cluster *mgmtv3.Cluster, dialer ty
 
 func notFound(err error) bool {
 	if awsErr, ok := err.(awserr.Error); ok {
-		return awsErr.Code() == eks.ErrCodeResourceNotFoundException
+		return awsErr.Code() == "ResourceNotFoundException"
```

Comment:
What is the downside of using a [const](https://github.com/aws/aws-sdk-go/blob/main/service/eks/errors.go#L77)  here?